docker/docker-compose-devel.yml

```diff
@@ -29,7 +29,6 @@ services:
       context: ..
       args: 
         - NETRC=$NETRC
-        - APPLICATION_ENV=development
```

Comment:
I moved this var below to the `environment` parameter, otherwise it would only be available to the image during the build process.